spec/spec_helper.rb

```diff
@@ -50,8 +50,10 @@
 require 'support/gridfs'
 
 RSpec.configure do |config|
-  config.color     = true
```

Comment:
Why is the color setting removed? Does the color actually increase the log size?

Reply:
It is specified in .rspec.

Reply:
Ah, okay